pygerduty/__init__.py

```diff
@@ -403,6 +416,11 @@ def resolve(self, requester_id):
     def acknowledge(self, requester_id):
         self._do_action('acknowledge', requester_id=requester_id)
 
+    def get_details_log_entry(self, requester_id):
```

Comment:
Yeah, thinking about it more I think it would make the most sense to have something like

```
def get_trigger_log_entry(self, **kwargs):
    match = TRIGGER_DETAILS_ID_RE.search(self.trigger_details_html_url)
    return self.log_entries.show(match.group("trigger_id"), **kwargs)
```

This way you get easy access to the triggering log event and still have control over the specific options you pass through.

Would that meet your needs?